test/integration/subnet_integration_test.go

```diff
@@ -57,3 +57,35 @@ func TestIntegrationSubnet_GetAndList(t *testing.T) {
 		t.Fatalf("Networks.Delete returned error %s\n", err)
 	}
 }
+
+func TestIntegrationSubnet_CRUD(t *testing.T) {
+	integrationTest(t)
+
+	autoCreateSubnet := false;
+	createNetworkRequest := &cloudscale.NetworkCreateRequest{
+		Name:                 networkBaseName,
+		AutoCreateIPV4Subnet: &autoCreateSubnet,
+	}
+	network, err := client.Networks.Create(context.TODO(), createNetworkRequest)
+	if err != nil {
+		t.Fatalf("Networks.Create returned error %s\n", err)
+	}
+
+	createSubnetRequest := &cloudscale.SubnetCreateRequest{
+		CIDR:    "192.168.192.0/22",
+		Network: network.UUID,
+	}
+	subnet, err := client.Subnets.Create(context.TODO(), createSubnetRequest)
+	if err != nil {
+		t.Fatalf("Subnets.Create returned error %s\n", err)
+	}
```

Comment:
We should probably assert here that the CIDR is 192...

Reply:
Also it would be nice to assert the network UUID. :)

Reply:
done